### PR TITLE
Print long error message in chunks

### PR DESCRIPTION
**Pull request by jonas-lj**

### CODE REVIEW

**Review by benr-ml — COMMENTED** [review]
Two quick comments:
- This is only one error that we print. We have other debug! calls that are also important.
- `.for_each(|x| debug!("{}", x.unwrap()))` will be hard to find in a very active log since we cannot match it to a specific tx id.

Perhaps let's do this logic in the debug! call sites?

**Comment by jonas-lj** [comment]
> Two quick comments:
> 
> * This is only one error that we print. We have other debug! calls that are also important.
> * `.for_each(|x| debug!("{}", x.unwrap()))` will be hard to find in a very active log since we cannot match it to a specific tx id.
> 
> Perhaps let's do this logic in the debug! call sites?

Regarding the first, I'll check the other places we've added logs to see if this is also an issue there.

Re the second: I see your point, but each debug message will actually show the current [debug span](https://docs.rs/tracing/latest/tracing/macro.debug_span.html) if such a span is defined. So, for example, if the call comes from a call stack originating in `process_transaction`, the span defined there, `aggregator_process_tx{tx_digest=TransactionDigest(XYZ)}`, will be included in each debug message.

File: crates/sui-types/src/crypto.rs [hunk 1]
```diff
@@ -1549,11 +1550,33 @@ impl<'a> VerificationObligation<'a> {
             pks,
             &self.messages.iter().map(|x| &x[..]).collect::<Vec<_>>()[..],
         )
-        .map_err(|e| SuiError::InvalidSignature {
-            error: format!(
+        .map_err(|e| {
+            let error_message = format!(
                 "Failed to batch verify aggregated auth sig {} pks: {:?} messages: {:?} sigs: {:?}",
                 e, &self.public_keys, &self.messages, &self.signatures
```
Comment (joyqvq) [inline crates/sui-types/src/crypto.rs:1556]: while you are at it, can we change this to base64 encoded bls aggregated sig? also messages can be encoded too.

**Review by benr-ml — APPROVED** [review]

**Review by kchalkias — APPROVED** [review]

File: crates/sui-types/src/crypto.rs [hunk 2]
```diff
@@ -1549,11 +1550,36 @@ impl<'a> VerificationObligation<'a> {
...
+                .as_bytes()
+                .chunks(chunk_size)
+                .map(std::str::from_utf8)
+                .enumerate()
+            {
+                debug!(
+                    "Failed to batch verify aggregated auth sig {} (chunk {}): {}",
+                    e,
+                    i,
+                    chunk.unwrap()
```
Comment (longbowlu) [inline crates/sui-types/src/crypto.rs:1576]: use `warn` instead. Node operators usually don't have `debug` level on

**Review by benr-ml — APPROVED** [review]

**Pull request closed without merging**